ccm_web/server/src/canvas/canvas.model.ts

```diff
@@ -0,0 +1,39 @@
+import { BelongsTo, Column, ForeignKey, Model, Table } from 'sequelize-typescript'
+import { DataTypes, Optional } from 'sequelize'
+
+import { User } from '../user/user.model'
+
+interface CanvasTokenAttributes {
+  id?: bigint
+  userId: bigint
+  accessToken: string
+  refreshToken: string
+}
+
+interface CanvasTokenCreationAttributes extends Optional<CanvasTokenAttributes, 'id'> {}
+
+@Table({
+  tableName: 'canvas_token',
```

Comment:
nice, this clearly defines what is the actual table name. use this in `session` table as well

Reply:
I added `session` to the `Session` model as the `tableName`. Should this still be `canvas_token` or did we want to change it to `canvasToken` or `CanvasToken`. Still unsure what the best practice would be in NodeJS land. The field names on `User` for instance are `firstName`, `lastName`, etc., as are the field names on `CanvasToken` currently.

Reply:
I would save stick with camel casing

Reply:
Okay, I'm going to assume you mean `canvasToken`.

Reply:
Done, see https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/92/commits/911f2cfb48289752ec97b81219534b3312414b4b